manifests/init.pp

```diff
@@ -64,8 +67,9 @@
   validate_hash($settings)
   validate_hash($ppas)
 
-  if $proxy['host'] {
+  if $proxy['ensure'] or $proxy['host'] {
```

Comment:
It's actually a bug on our side right now, https://github.com/puppetlabs/puppetlabs-apt/blob/master/manifests/init.pp#L67-L72 doesn't use `$_proxy` but it should have.